demo-certs/certs/coap-adapter-cert.pem

```diff
@@ -0,0 +1,64 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
has this cert been added to the existing certs, i.e. has it been created/signed using the *existing* intermediate cert or have you re-created *all* keys and certificates by running the full create_certs script?

Reply:
I hope, I used the existing certs :-).
At least, using this certs and to already available other certs, allows to connect the coap-adapter to the qpid-router.